joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -46,15 +78,33 @@ trait AstForTypesCreator(implicit withSchemaValidation: ValidationMode) { this:
       alias = None
     )
 
+    val metaTypeDeclTemp = typeDeclNode(
```

Comment:
This does not seem to be a temp variable because it ends up in the resulting AST. Please rename.